crates/rosty/src/lib.rs

```diff
@@ -40,6 +42,11 @@ pub async fn init_with_args(args: NodeArgs, capture_sigint: bool) -> Result<(),
 
     let node = Node::new(args).await?;
 
+    // Initialize the use of simtime
+    if node.is_using_sim_time() {
```

Comment:
Why not do this in Node::new?

Reply:
I don't know, because it is so stateful if feels weird putting it there?